test/integrate/dubbo/go-server/go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/apache/dubbo-go/test/integrate/dubbo/go-server
+
+go 1.14
```

Comment:
in main package go version is 1.13 , should u change it? if not , may cause compile problem.

https://github.com/apache/dubbo-go/blob/master/go.mod

Reply:
Okay, I have modified it in the latest commit.